src/main/java/io/candydoc/ddd/tactical_pattern/Factory.java

```diff
@@ -0,0 +1,26 @@
+package io.candydoc.ddd.tactical_pattern;
+
+import io.candydoc.ddd.model.DDDPattern;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class Factory extends TacticalModel {
+
+  private static final Set<DDDPattern> ALLOWED_DEPENDENCIES =
+      Set.of(TacticalPattern.AGGREGATE, TacticalPattern.VALUE_OBJECT, TacticalPattern.FACTORY);
```

Comment:
I think it can also depend on domain entity

Reply:
Yes you're right, we add it right now